app/helpers/eu_exit_finder_helper.rb

```diff
@@ -0,0 +1,9 @@
+module EuExitFinderHelper
+  def self.eu_exit_finder?(content_id)
+    content_id == "42ce66de-04f3-4192-bf31-8394538e0734"
```

Comment:
Thanks Steve - do you mean [like this](http://www.chrisrolle.com/en/blog/private-constants-in-ruby):

```ruby
module EuExitFinderHelper
  CONTENT_ID_EU_EXIT_FINDER = "42ce66de-04f3-4192-bf31-8394538e0734"
  CONTENT_ID_EU_EXIT_FINDER_EMAIL_SIGNUP = "2818d67a-029a-4899-a438-a543d5c6a20d"
  private_constant :CONTENT_ID_EU_EXIT_FINDER
  private_constant :CONTENT_ID_EU_EXIT_FINDER_EMAIL_SIGNUP

  def self.eu_exit_finder?(content_id)
    content_id == CONTENT_ID_EU_EXIT_FINDER
  end

  def self.eu_exit_finder_email_signup?(content_id)
    content_id == CONTENT_ID_EU_EXIT_FINDER_EMAIL_SIGNUP
  end
end
```

...or [like this](https://www.honeybadger.io/blog/when-to-use-freeze-and-frozen-in-ruby/):

```ruby
module EuExitFinderHelper
  def self.eu_exit_finder?(content_id)
    content_id == "42ce66de-04f3-4192-bf31-8394538e0734".freeze
  end

  def self.eu_exit_finder_email_signup?(content_id)
    content_id == "2818d67a-029a-4899-a438-a543d5c6a20d".freeze
  end
```

Reply:
The first way means a string is only created when once when the module is loaded whereas defining it in a method means it's recreated for each call, it's an almost insignificant performance consideration. It's not like the method is called when looping through millions of records.

Reply:
Interesting. I suppose if this _were_ in a big loop then it would be a change worth making. As it is, feels to me like option 1 sacrifices code readability for little benefit! And option 2, while readable, is probably pointless (don't see how anyone could alter the value of the hardcoded string from outside the method)

What do you think? Shall I leave it?